File: Python/Single Responsibility Principle/Adres.py
class Adres:
    def __init__(self):
        self.ulke = None
        self.il = None
        self.ilce = None
        self.postkodu = None

    def getulke(self):
        return self.ulke

    def setulke(self, ulke):
        self.ulke = ulke

    def getil(self):
        return self.il

    def setil(self, il):
        self.il = il

    def getilce(self):
        return self.ilce

    def setilce(self, ilce):
        self.ilce = ilce

    def getpostkodu(self):
        return self.postkodu

    def setpostkodu(self, postkodu):
        self.postkodu = postkodu
        
    def __str__(self):
        return f"{self.ulke} {self.il} {self.ilce} {self.postkodu}"

File: Java/Single Responsibility Principle/Adres.java
public class Adres {
    private String Ulke;
    private String il;
    private String ilce;
    private int postaKodu;

    public Adres(String Ulke, String il, String ilce,int postaKodu){
        this.Ulke=Ulke;
        this.il=il;
        this.ilce=ilce;
        this.postaKodu=postaKodu;

    }
    @Override
    //string ifadeyi ekrana yazdırma amacıyla
    public String toString(){
        return (Ulke+" "+il+" "+ilce+" "+postaKodu);
    }
    //set fonk eklenecek
    public String getulke(){
        return Ulke;
    }

    public String getil(){
        return il;
    }

    public String getilce(){
        return ilce;
    }

    public int getpostaKodu(){
        return postaKodu;
    }
}   
